src/06-extensions.md

```diff
@@ -47,7 +47,7 @@ provides a direct link to the documentation.
 | [BEP019](https://docs.google.com/document/d/1FqJI791ycXr0bfRg2qyLqAf0RpVttJ2cInOgMWrKsNU) | DICOM Metadata                                                                                             | Satrajit Ghosh                                                             | Not active - perhaps this is in the 20% of use cases. Could be a NIDM extension rather than in BIDS.                                                                                                                                      | None.                                                                                  |
 | [BEP020](https://docs.google.com/document/d/1eggzTCzSHG3AEKhtnEDbcdk-2avXN6I94X8aUPEBVsw) | Eye Tracking including Gaze Position and Pupil Size(ET)                                                    | Benjamin Gagl and Dejan Draschkow                                          | BEP is progressing and reaching consensus on items.                                                                                                                                                                                       | Small issues are blocking completion.                                                  |
-| [BEP021](https://docs.google.com/document/d/1PmcVs7vg7Th-cGC-UrX8rAhKUHIzOI-uIOh69_mvdlw) | Common Electrophysiological Derivatives                                                                    | Mainak Jas, Stefan Appelhoff, Cyril Pernet, Robert Oostenveld, Teon Brooks | Work has resumed since merging in EEG and iEEG. Next steps are to distribute a survey to gather what derivatives researchers care about, create example datasets to understand practical issues, and updates to the validator.            | A few blocking items is what file format may be best suited and annotations.           |
+| [BEP021](https://docs.google.com/document/d/1PmcVs7vg7Th-cGC-UrX8rAhKUHIzOI-uIOh69_mvdlw) | Common Electrophysiological Derivatives                                                                    | Mainak Jas, Stefan Appelhoff, Cyril Pernet, Robert Oostenveld | Work has resumed since merging in EEG and iEEG. Next steps are to distribute a survey to gather what derivatives researchers care about, create example datasets to understand practical issues, and updates to the validator.            | A few blocking items is what file format may be best suited and annotations.           |
```

Comment:
```suggestion
| [BEP021](https://docs.google.com/document/d/1PmcVs7vg7Th-cGC-UrX8rAhKUHIzOI-uIOh69_mvdlw) | Common Electrophysiological Derivatives                                                                    | Mainak Jas, Stefan Appelhoff, Cyril Pernet, Robert Oostenveld              | Work has resumed since merging in EEG and iEEG. Next steps are to distribute a survey to gather what derivatives researchers care about, create example datasets to understand practical issues, and updates to the validator.            | A few blocking items is what file format may be best suited and annotations.           |
```